client/app/components/composites/AvatarDropdown/AvatarDropdown.js

```diff
@@ -37,13 +64,16 @@ class AvatarDropdown extends Component {
   }
 
   render() {
-    const openOnHoverClass = hasTouchEvents ? '' : css.openOnHover;
+    const openOnHoverClass = this.state.isMounted ? '' : css.openOnHover;
+    const transitionDelayClass = this.state.isMounted ? '' : css.transitionDelay;
     const openClass = this.state.isOpen ? css.openDropdown : '';
     const notificationsClass = this.props.notificationCount > 0 ? css.hasNotifications : null;
     const notificationBadgeInArray = this.props.notificationCount > 0 ?
       [r(NotificationBadge, { className: css.notificationBadge }, this.props.notificationCount)] :
       [];
     return div({
+      onMouseOver: this.handleMouseOver,
+      onMouseLeave: this.handleMouseOut,
```

Comment:
Maybe the method could be renamed to `handleMouseLeave` to indicate the mouse event it attaches to?